includes/admin/class-wc-rest-payments-disputes-controller.php

```diff
@@ -74,9 +83,20 @@ public function register_routes() {
 
 	/**
 	 * Retrieve disputes to respond with via API.
+	 *
+	 * @param WP_REST_Request $request Full data about the request.
+	 */
+	public function get_disputes( WP_REST_Request $request ) {
+		$page      = (int) $request->get_param( 'page' );
+		$page_size = (int) $request->get_param( 'pagesize' );
+		return $this->forward_request( 'list_disputes', [ $page, $page_size ] );
+	}
+
+	/**
+	 * Retrieve transactions summary to respond with via API.
 	 */
-	public function get_disputes() {
-		return $this->forward_request( 'list_disputes', [] );
+	public function get_disputes_summary() {
```

Comment:
We should doc-block the return type here (like in `get_dispute`). We also probably accept request object as an argument here, or?